### ISSUE TITLE
Make storage list as `list` output

### ISSUE THREAD

**Issue by surajnarwade** [redhat-developer/ocdev#285]
Currently `ocdev storage list` output shows following output:

```
$ ocdev storage list
deploymentconfigs/nodejs
  pvc/pvc-x9kkn (allocated 100GiB) as nodejs
    mounted at /home

```

But it should give list output so that it will be easy to use it in description in application and component and for better UX.

**Comment by containscafeine** [comment]
This is blocking #278 
I'm working on this, and will then move over to the other storage commands.
Will start by moving storage to client-go

**Comment by containscafeine** [comment]
Fixed in #295
